File: Src/java/engine/src/main/java/org/opencds/cqf/cql/engine/elm/executing/TupleEvaluator.java
package org.opencds.cqf.cql.engine.elm.executing;

import java.util.LinkedHashMap;
import org.opencds.cqf.cql.engine.execution.State;

public class TupleEvaluator {

    public static Object internalEvaluate(LinkedHashMap<String, Object> ret, State state) {
        return new org.opencds.cqf.cql.engine.runtime.Tuple(state).withElements(ret);
    }
}

File: Src/java/engine/src/main/kotlin/org/opencds/cqf/cql/engine/elm/executing/TupleEvaluator.kt
package org.opencds.cqf.cql.engine.elm.executing

import org.opencds.cqf.cql.engine.runtime.Tuple

object TupleEvaluator {
    fun internalEvaluate(ret: MutableMap<String, Any?>?): Tuple {
        return Tuple().withElements(ret!!)
    }
}
